examples/inference/build_gptq_weight.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import logging
+
+from auto_gptq import AutoGPTQForCausalLM, BaseQuantizeConfig
+from transformers import AutoTokenizer
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)s [%(name)s] %(message)s", level=logging.INFO, datefmt="%Y-%m-%d %H:%M:%S"
+)
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--pretrained_model_dir",
+    type=str,
+    help="pretrained model directory",
+)
+parser.add_argument(
+    "--quantized_model_dir",
+    help="the path of out gptq model",
+    type=str,
+)
+args = parser.parse_args()
+pretrained_model_dir = args.pretrained_model_dir
+quantized_model_dir = args.quantized_model_dir
+
+
+quantize_config = BaseQuantizeConfig(
+    bits=4,  # quantize model to 4-bit
+    group_size=128,  # it is recommended to set the value to 128
+    desc_act=False,  # set to False can significantly speed up inference but the perplexity may slightly bad
+)
+
+tokenizer = AutoTokenizer.from_pretrained(pretrained_model_dir)
+examples = [
+    tokenizer(
```

Comment:
How should we choose examples and what's the effect of them?

Reply:
The examples can affect the accuracy of gptq model. You can replace the examples with your own data.

Reply:
This is hard for users to choose. Can we provide a general dataset or a guide?

Reply:
This is a simple example for benchmark. Users can look more information from AutoGPTQ https://github.com/PanQiWei/AutoGPTQ

Reply:
No, as we provide this feature, we have to instruct users how to use it.

Reply:
Users can replace their own data based on provided examples. There is no difference  that we provide another datasets.

Reply:
How does a beginner know the method to choose this? As for me, after reading this file, I don't know how to choose a proper dataset.

Reply:
Add example dataset and dataset path.